### DISCUSSION TITLE
Watch vs P2G

### DISCUSSION THREAD

**Discussion by the-mace** [philosowaffle/peloton-to-garmin#625] [Q&A]
Hi folks, during the winter I do a bunch of cycling on the Peloton and sync those results to Garmin Connect. I finally got a Garmin watch (F7s pro SS) and used it today as well as the Peloton to track my exercise. Both were using the Garmin heart rate monitoring on the watch.
Peloton reported 537 calories used
Garmin reported 285 calories used.
Peloton knows speed and power whereas the Garmin doesn't.
Does this explain the difference? It's almost 2x more for Peloton.

What are people doing when they have both options? It doesnt seem like things merge at garmin connect.

Is it always best to use the "original" source?

I have a similar question for when I start outdoor riding again. I have a Garmin Edge Explore 2. I think there I want to track **only** on the Edge Explore 2 since it will also get cadence etc. and capture heart rate from the watch, but does that miss advanced watch/health analytics?

**Comment by Vicore1** [comment]
`What are people doing when they have both options? ` We use the watch for heat rate we delete the activity when we are done and use the sync from peloton.
`Is it always best to use the "original" source?`
I would use whatever has more data points, to generate that calories burned if that's the source (peloton or a bike mounted computer) I would use that. You should still get those same analytics in your app/ on your watch.

**Reply by the-mace** [reply]
Thank you, I was thinking the answer may be. So the watch for me would be more for workouts where its the only source like running, hiking, walking etc.

**Reply by Vicore1** [reply]
That is exactly how I use mine and that is the recommended use case.

**Reply by the-mace** [reply]
Thanks just making sure I wasnt thinking through this the wrong way.

**Comment by jheizer** [comment]
A lot of the watches have just a broadcast HR mode.  Turn that on and then all the data is in the peloton activity.

**Reply by the-mace** [reply]
Yeah that's how I rode this am (I used to ride with a Scoche Rhythm+ before the new Fenix arrived). I was more interested if this is just explained by more data being on the peloton (speed, cadence) vs inaccuracies of Peloton vs Garmin

**Reply by Vicore1** [reply]
It's the resistance. The additional resistance is the reason for the additional calories. For outdoor riding.,It would be changing altitude that would be the reason for additional calories vs stationary. Without that metric being factored in the watch basically assumed (because it doesn't have that metric) a resistance/ change of altitude of zero.

**Reply by the-mace** [reply]
That makes sense. I've also been looking at power meters for the outdoor bike but they're messy and expensive.

**Reply by Vicore1** [reply]
I'm a lazy exerciser so I don't do any outdoor biking, running, rowing, but I imagine with the GPS functionality for altitude along with speed and cadence you'll get a pretty accurate calorie burn number. That's based on my experience hiking where shift in altitude factors in.
